tests/unit/test_main.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import sys
+
+from unittest.mock import patch, Mock
+
+from auditwheel.main import main
+
+
+def test_non_linux_platform(monkeypatch, capsys):
+    # GIVEN
+    monkeypatch.setattr(sys, "platform", "darwin")
```

Comment:
As mentioned by @takluyver, what if `darwin` is supported ? I'd say `not_a_platform`